cortex/svgoverlay.py

```diff
@@ -641,7 +645,7 @@ def make_svg(pts, polys):
     return svg
 
 def get_overlay(subject, svgfile, pts, polys, remove_medial=False, 
-                overlays_available=None, **kwargs):
+                overlays_available=None, allow_change=True, **kwargs):
     """Return a python represent of the overlays present in `svgfile`
```

Comment:
I know this function doesn't really have a docstring, but I find the keyword `allow_change` a bit cryptic. Can you add a note about it in the docstring, or maybe rename it to `modify_svg_file`?